unison-src/transcripts/propagate.output.md

```diff
@@ -175,93 +175,3 @@ scratch/main> view preserve.otherTerm
   preserve.otherTerm y = someTerm y
 
 ```
-### Propagation only applies to the local branch
```

Comment:
This is inexpressable now that we can't cd into a subnamespace of a project branch.